gem/lib/metasploit-payloads.rb

```diff
@@ -88,6 +168,14 @@ def self.list_meterpreter_extension_suffixes(extension_name=nil)
     list_meterpreter_dirs { |dir| meterpreter_enum_ext_suffixes(dir, extension_name) }
   end
 
+  def self.manifest_path
+    ::File.realpath(::File.join(::File.dirname(__FILE__), '..', 'manifest'))
+  end
+
+  def self.manifest_uuid_path
+    ::File.realpath(::File.join(::File.dirname(__FILE__), '..', 'manifest.uuid'))
+  end
```

Comment:
It'd be great if this was private, if it's easy enough to add it to the eigenclass beneath and the tests still pass - that works for me.

If not, we can circle back and make it private